php-frontend/src/main/java/org/sonar/php/cfg/ControlFlowGraphBuilder.java

```diff
@@ -135,8 +137,26 @@ private PhpCfgBlock buildBlock(BlockTree block, PhpCfgBlock successor) {
   }
 
   private PhpCfgBlock buildIfStatement(IfStatementTree tree, PhpCfgBlock successor) {
+    PhpCfgBlock elseBlock = successor;
+    if (tree.elseClause() != null) {
+      elseBlock = buildSubFlow(tree.elseClause().statements(), successor);
+    }
+    PhpCfgBlock falseSuccessor = elseBlock;
```

Comment:
drop `elseBlock` var, use `falseSuccessor` from start (and rename to `falseBlock` to be consistent)